person.rb

```diff
@@ -29,6 +30,11 @@ def print_of_age
   def of_age?
     @age >= 18
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
I suggest the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association😉.

```suggestion
   def add_rental(book, date)
    Rental.new(date, book, self)
  end
```
```

Reply:
Hi @Ibizugbe ,

Thanks for your time and for reviewing my code and suggesting the solution. I will apply the all the required changes. 🙏